packages/react-select/src/components/Group.tsx

```diff
@@ -101,6 +103,7 @@ interface GroupHeadingPropsDefinedProps<
   selectProps: Props<Option, IsMulti, Group>;
   theme: Theme;
   getStyles: GetStyles<Option, IsMulti, Group>;
+  getClassName: CommonProps<Option, IsMulti, Group>['getClassName'];
```

Comment:
Not sure why these types are re-defined here, but just following the existing pattern to simplify this PR